app/views/prison/visits/show.html.erb

```diff
@@ -13,8 +13,11 @@
         <div class="tag tag--<%= @visit.processing_state %> tag--heading"><%= @visit.processing_state.capitalize %>: <br><%= @visit.processed_at.to_date.to_s(:short_nomis) %> - <%= @visit.processed_at.to_s(:time) %></div>
       </div>
       <div class="column-one-third">
+        <% if @visit.human_id -%>
```

Comment:
All visits have human ids in production so the check can go. You might need to generate human ids for your local database